src/Quantity.js

```diff
@@ -1,8 +0,0 @@
-import {Record} from 'immutable'
```

Comment:
Please also remove this:

https://github.com/ePages-de/immutable-api-records/blob/50375fc1bd58b8eceabbf00b4b33f01e1c229f70/src/index.js#L71